molgenis-data-csv/src/main/java/org/molgenis/data/csv/CsvIterator.java

```diff
@@ -149,6 +151,13 @@ private Entity get()
 						next.set(name, valueList.get(colNamesMap.get(name)));
 					}
 				}
+				else if (values != null && (values.length > 1 || (values.length == 1 && values[0].length() > 0))
+						&& values.length < colNamesMap.size())
```

Comment:
Unsure what is happening here. A single column containing the empty string is allowed?

Reply:
Yes, e.g. col1,col2\n\n

Reply:
I'd say just forbid those empty lines as well?

Reply:
Disallowing trailing empty lines will break imports of existing data, I suggest we do not do this as part of the issue being fixed.